docs/README.md

```diff
@@ -0,0 +1,40 @@
+# Amazon Cloud Controller Manager
```

Comment:
@mcrute why did you close this?
could you at least put this file in so it doesn't mislead people that this is currently usable?